explainers/subresource-loading.md

```diff
@@ -156,6 +156,8 @@ subresources, `a.js`, `b.js`, and `c.png`, from the bundle.
 `urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6` is also loaded from
 the bundle, and a subframe is instantiated as an
 [opaque origin](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque) frame.
+The browser must ignore any [registered custom protocol handler](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-registerprotocolhandler) in loading a resource
+from a bundle.
```

Comment:
This is much more comfortable; thanks @hayatoito. Custom protocol handlers are always used from https://html.spec.whatwg.org/multipage/browsing-the-web.html#process-a-navigate-url-scheme, which looks like it's only called after filtering out "fetch schemes" (like `https:`). Since a URL in a bundle is fetchable, it makes sense to avoid checking custom protocol handlers. So ...

```suggestion
If a URL is available from an attached bundle, the browser must retrieve it from the bundle, instead of using any [registered custom protocol handler](https://html.spec.whatwg.org/multipage/system-state.html#dom-navigator-registerprotocolhandler) for its scheme.
```

Reply:
Done